README.nob.md

```diff
@@ -10,9 +10,15 @@ Denne guiden på andre språk:
 
 En applikasjon som simulerer mandatdistribusjon i valg. Nåværende funksjonalitet inkluderer:
 
--   Stortingsvalgene i 2017, 2013 og 2009
--   Fordelingsmetodene Sainte-Lagüe (modifisert) og D'Hondt
+-   Alle stortingsvalg siden 1945
+-   Fordelingsmetodene Sainte-Lagüe (modifisert), d'Hondt, største brøks metode (Hare, Droop & Hagenbach-Bischoff)
 -   Detaljerte tabeller som beskriver simulasjonsresultatene
+-   Parametere for å justere hvordan mandatene distribueres
+-   Sammenslåing av fylker for å simulere effekten av de nye fylkene i 2020
```

Comment:
```suggestion
-   Sammenslåing av fylker for å simulere effekten av de nye regionene i 2020
```
might be an improvement

Reply:
They are counties, rather than regions. Isn't it more correct to use the proper word for it?

Reply:
It is more clear it has to do with the region reform this way (and it is unknown whether the elections will work this way in the future).